stdlib/public/Concurrency/Task.swift

```diff
@@ -17,15 +17,23 @@ import Swift
 
 /// A unit of asynchronous work.
 ///
-/// An instance of `Task` always represents a top-level task. The instance
-/// can be used to await its completion, cancel the task, etc., The task will
-/// run to completion even if there are no other instances of the `Task`.
+/// When you create an instance of `Task`,
+/// you provide a closure that contains the work for that task to perform.
+/// Tasks can start running immediately after creation;
+/// you don't explicitly start or schedule them.
+/// After creating a task, you use the instance to interact with it ---
+/// for example, to wait for it to complete or to cancel in.
+/// It's not a programming error to discard a reference to a task
+/// without waiting for that task to finish or canceling it.
+/// A task runs whether or not you keep a reference to it.
+/// However, if you discard a task's handle, you give up the ability
```

Comment:
You seem to move wording to use "task reference" so perhaps also remove the "handle" word here?

```suggestion
/// However, if you discard a task reference, you give up the ability
```

Reply:
Changed in 946c72cbc73, thanks.